src/SwedbankPay.Sdk/PaymentInstruments/Card/CardPaymentCardDetails.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    /// <summary>
+    /// Object to hold details about a payments card details.
+    /// </summary>
+    public class CardPaymentCardDetails : ICardPaymentCardDetails

```

Comment:
I've commented this above and think that this should be renamed to `CardPaymentAuthorizationTransaction`.

```suggestion
    public class CardPaymentAuthorizationTransaction : ICardPaymentAuthorizationTransaction
```